tutorial_sarscov2.md

```diff
@@ -0,0 +1,259 @@
+
+You can use the mouse selection to zoom in region and double clicks to zoom out.
+
+### Expected output
+
+The small dataset that we used in this tutorial section has been analyzed by [doi:10.1093/nsr/nwaa036](https://doi.org/10.1093/nsr/nwaa036).
+The results of the original analysis (using bwa, samtools mpileup, and bcftools) are displayed in Table 2 in the article:
+
+|Accession number|Genomic position|Ref allele|Alt allele|Ref reads|Alt reads|Location_date  |GISAID ID     |
+|:---------------+---------------:+:--------:+:--------:+--------:+--------:+:--------------+:-------------|
+|SRR10903401     |            1821|     G    |     A    |       52|        5|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903401     |           19164|     C    |     T    |       40|       12|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903401     |           24323|     A    |     C    |      102|       67|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903401     |           26314|     G    |     A    |       15|        2|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903401     |           26590|     T    |     C    |       10|        2|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903402     |           11563|     C    |     T    |      164|       26|WH_2020/01/02.b|EPI_ISL_406717|
+
+Using the reports and zoom function, compare with the results given out by V-pipe (with bwa and ShoRAH).
+
+- For positions 19164 and 24323 of SRR10903401 and position 11563 of SRR10903402,
+  we expect to see similar results in V-pipe.
+- For the remaining positions (1821, 26314 and 26590 of SRR10903401),
+  we expect that ShoRAH will consider the variants of poor quality and reject them
+  because there is very little support ( \<=  than 5 reads supporting the alt).
+
+## Swapping component
+
+The default configuration uses [ShoRAH](https://cbg-ethz.github.io/shorah/)
+to call the SNVs and to reconstruct the local (windowed) haplotypes.
+
+Components can be swapped simply by changing the `vpipe.config` file.
+For example to call SNVs using `lofreq`:
+
+```
+[output]
+snv = True
+local = False
+
+[general]
+snv_caller=lofreq
+```
+
+
+## Cluster deployment
+
+It is possible to ask snakemake to [submit jobs on a cluster](https://snakemake.readthedocs.io/en/stable/executing/cluster-cloud.html#cluster-execution) using the batch submission command-line interface of your cluster.
+
+[Platform LSF by IBM](https://www.ibm.com/support/knowledgecenter/SSETD4_9.1.2/lsf_command_ref/bsub.1.html) is one of the popular systems you might find (Others include [SLURM](https://slurm.schedmd.com/sbatch.html), [Grid Engine](https://en.wikipedia.org/wiki/Oracle_Grid_Engine)).
+
+*[LSF]: Load Sharing Facility
+
+To deploy on the cluster:
+
+```bash
+wget 'https://raw.githubusercontent.com/cbg-ethz/V-pipe/master/deploy_vpipe.sh'
+bash deploy_vpipe.sh -b sars-cov2 -p $SCRATCH -w working
+cd $SCRATCH/working/
+```
+
+ - using `-p` help us store V-pipe on some large-storage share, e.g.: scratch
```

Comment:
Is this a good idea? I guess you meant to place the working directory on scratch.

Reply:
Given the size of conda installation, I wanted to point out this possibility to the users.

Of course, 
`bash quick_install.sh -b sars-cov2 -p $HOME/virology_project/ -w $SCRATCH/working`
does also work.

Your pick, at this point.

Reply:
If you install it in scratch it will be wipe out within 15 days in Euler which is already very nice. Other cluster admins can do it differently. 
For a single-use, it is probably fine, e.g., the testing exercise. But for a user that intends to use the pipeline for a long while, this is not going to work